search/query/query_string.y

```diff
@@ -127,7 +127,12 @@ tSTRING tCOLON tSTRING tTILDE {
 tNUMBER {
 	str := $1
 	logDebugGrammar("STRING - %s", str)
-	q := NewMatchQuery(str)
+	q1 := NewMatchQuery(str)
+	val, _ := strconv.ParseFloat($1, 64)
```

Comment:
If err handling/propagation supported in these foo.y files?  I guess ParseFloat() might be able to return a range/syntax error, even if it looks numeric'ish from the parser's point of view...

https://golang.org/src/strconv/atof.go?s=11079:11134#L523